src/unix/newlib/mod.rs

```diff
@@ -557,16 +585,21 @@ pub const TCP_KEEPCNT: ::c_int = 1024;
 cfg_if! {
     if #[cfg(target_os = "horizon")] {
         pub const IP_TOS: ::c_int = 7;
```

Comment:
All of these have wrong values for ESP-IDF, but I must say this is not due to this PR. It was also wrong before, and needs fixing: 
* https://github.com/espressif/esp-lwip/blob/7896c6cad020d17a986f7e850f603e084e319328/src/include/lwip/sockets.h#L271
* https://github.com/espressif/esp-lwip/blob/7896c6cad020d17a986f7e850f603e084e319328/src/include/lwip/sockets.h#L307